docs/installation.rst

```diff
@@ -8,38 +8,37 @@ Installation
 
 .. code-block:: bash
 
-    $ git clone https://github.com/BeyondTheClouds/enos
-
-You should also choose to go with a virtualenv. Please refer to the `virtualenv
-<https://virtualenv.pypa.io/en/stable/>`_ documentation for further information.
+    $ pip install enos
```

Comment:
Should I have to change the `reservation.yaml.sample` to reflect the fact that enos:master is in sync with kolla-ansible:master? Because, this is not currently the case if I am right.